opsdroid/database/mongo/tests/test_mongo.py

```diff
@@ -35,49 +52,40 @@ async def test_connect(database):
 
 
 @pytest.mark.asyncio
-@pytest.mark.parametrize("config", [{}])
-async def test_get(database):
-    database.database = {}
-    database.database["test_db"] = DatabaseMongoTest({})
-    with pytest.raises(TypeError):
-        await database.get("test_db")
+@pytest.mark.parametrize("config", [{"collection": "test_collection"}])
+async def test_get(mocked_database):
+    mocked_database.database = {"test_collection": DatabaseMongoCollectionMock({})}
```

Comment:
You don't need this line because it is handled in the `mocked_database` fixture.
```suggestion
```

Reply:
You right, you right. Updated.
Ok, my friend @eheiberg helped me create a test for the context manager. We think we're at a pretty good place.

Reply:
The more I look at it, I'm not sure there is anything to assert in the general case. Since we are mocking the mongo database/collection, I think we are mostly looking to make sure nobody adds any code that would generate an exception with the test as defined. Let me know if you see something different.